Can't enable communities anymore

As of latest master (`362ad02f`) I'm unable to login with existing accounts that are part of communities, or, when using a brand new account, turning on communities feature crashes the app with

```
Traceback (most recent call last)
/Users/pascalprecht/projects/status-im/nim-status-client/src/nim_status_client.nim(203) nim_status_client
/Users/pascalprecht/projects/status-im/nim-status-client/src/nim_status_client.nim(200) mainProc
/Users/pascalprecht/projects/status-im/nim-status-client/vendor/nimqml/src/nimqml/private/qapplication.nim(8) exec
/Users/pascalprecht/projects/status-im/nim-status-client/vendor/nimqml/src/nimqml/private/qobject.nim(49) qobjectCallback
/Users/pascalprecht/projects/status-im/nim-status-client/vendor/nimqml/src/nimqml/private/qobject.nim(34) onSlotCalled
/Users/pascalprecht/projects/status-im/nim-status-client/vendor/nimbus-build-system/vendor/Nim/lib/core/macros.nim(567) onSlotCalled
/Users/pascalprecht/projects/status-im/nim-status-client/src/app/chat/views/communities.nim(119) getJoinedComunities
/Users/pascalprecht/projects/status-im/nim-status-client/src/status/chat.nim(465) myPendingRequestsToJoin
/Users/pascalprecht/projects/status-im/nim-status-client/src/status/libstatus/chat.nim(379) myPendingRequestsToJoin
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [run-macos] Error 1
```

--- a/src/status/libstatus/chat.nim
+++ b/src/status/libstatus/chat.nim
@@ -376,7 +376,7 @@ proc myPendingRequestsToJoin*(): seq[CommunityMembershipRequest] =
   let rpcResult = callPrivateRPC("myPendingRequestsToJoin".prefix).parseJSON()
   var communityRequests: seq[CommunityMembershipRequest] = @[]
 
-  if rpcResult{"error"}.kind == JNull and rpcResult{"result"}.kind != JNull:
+  if rpcResult.hasKey("result") and rpcResult{"result"}.kind != JNull:
     for jsonCommunityReqest in rpcResult["result"]:
       communityRequests.add(jsonCommunityReqest.toCommunityMembershipRequest())
 

